openedx/core/djangoapps/catalog/utils.py

```diff
@@ -98,7 +102,14 @@ def check_catalog_integration_and_get_user(error_message_field):
 
 
 # pylint: disable=redefined-outer-name
-def get_programs(site=None, uuid=None, uuids=None, course=None, catalog_course_uuid=None, organization=None):
+def get_programs(
+    site: "Site" = None,
```

Comment:
if a type is only imported for type checks, putting it in quotation marks, as here, will allow the type checker to do its thing without actually causing an import error during runtime.